inst/app/www/reportDocx.Rmd

```diff
@@ -117,6 +118,10 @@ tagList(br(), h2("Community Usage Metrics"))
 
 
 ```{r community_metrics_table, warning=FALSE, message=FALSE, error=FALSE, results='HIDE', echo=FALSE}
+if (!cm_ind) {
+    h6(glue::glue("Community Usage Metrics not avaiable for {params$pkg$name}"),
```

Comment:
This message doesn't display! It may be due to the code chunk option `results='HIDE'`?

word docx:
![image](https://user-images.githubusercontent.com/17878953/202466104-e60fd008-02aa-4daf-85f4-9de74e7922f7.png)

Reply:
I'm not sure what is happening. The line displays for me. Although its formatting may not be desirable.

![image](https://user-images.githubusercontent.com/85879620/202487077-bb946ae4-539c-45e2-b13a-7803b6502d02.png)